TestCases/serial_regression.py

```diff
@@ -93,8 +93,8 @@ def main():
     fixedCL_naca0012           = TestCase('fixedcl_naca0012')
     fixedCL_naca0012.cfg_dir   = "fixed_cl/naca0012"
     fixedCL_naca0012.cfg_file  = "inv_NACA0012.cfg"
-    fixedCL_naca0012.test_iter = 500
-    fixedCL_naca0012.test_vals = [-5.856803,-0.469459,0.299943,0.019385] #last 4 columns
+    fixedCL_naca0012.test_iter = 100
```

Comment:
You may have already checked this, but does 100 iterations cover at least one change of the AoA with fixed Cl mode? Just want to make sure we're covering that.

Reply:
I didn't think of this ... but fortunately in the config file the UPDATE_ALPHA is set to 5, so it should be covered.

Reply:
I think that is actually the number of times to update over the length of the calculation. Can you please double check this? I think we can just add a few iterations to this one if necessary...

Reply:
I just checked, the interval when alpha is updated is computed as EXT_ITER/UPDATE_ALPHA in the code.